src/Agent.Worker/TestResults/JunitResultReader.cs

```diff
@@ -330,6 +332,25 @@ private void ProcessFailureNode(XmlNode failure, TestCaseResultData resultCreate
             }
         }
 
+        private void AddSystemLogsToResult(XmlNode testCaseNode, TestCaseResultData resultCreateModel)
+        {
+            XmlNode stdout, stderr;
+
+            // Standard output logs
+            stdout = testCaseNode.SelectSingleNode("./system-out");
+            if (stdout != null && string.IsNullOrWhiteSpace(stdout.InnerText) == false)
```

Comment:
instead of `string.IsNullOrWhiteSpace(stdout.InnerText) == false` you can just do `!string.IsNullOrWhiteSpace(stdout.InnerText)` ?

Reply:
Done. Thanks